src/Admin/AbstractAdmin.php

```diff
@@ -1614,19 +1597,19 @@ public function removeShowFieldDescription($name): void
         unset($this->showFieldDescriptions[$name]);
     }
 
-    public function getListFieldDescriptions()
+    public function getListFieldDescriptions(): array
     {
         $this->buildList();
 
         return $this->listFieldDescriptions;
     }
 
-    public function getListFieldDescription($name)
+    public function getListFieldDescription($name): FieldDescriptionInterface
```

Comment:
```suggestion
    public function getListFieldDescription(string $name): FieldDescriptionInterface
```

Reply:
It's part of the **TODO** list:
>Check if is possible to perform similar update for method arguments;

By the moment, this PR is only focused on return declarations. I'd like to do the same with method arguments in an automated way.